.github/ISSUE_TEMPLATE/DISCUSSION.md

```diff
@@ -0,0 +1,31 @@
+---
+name: 💬 Discussion
+about: Create a discussion topic.
+labels: 'note: to discuss'
+---
+
+<!--
+Hello 👋 Thank you for submitting a discussion topic.
+
+To make your topic readable make sure you use valid Markdown syntax.
+
+https://guides.github.com/features/mastering-markdown/
+-->
+
+**Describe the topic**
+A clear and concise description of what the topic is.
```

Comment:
I would also add: `Please provide the detailed steps to reproduce your issue.`

Reply:
@soupette It may not be an issue, it may be a design discussion in which there isn't going to be any steps related to it, this template is more so for discussing things instead of just using the slack (which only has limited chat storage)

Reply:
You are right.